capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -50,17 +50,16 @@ export class WikiMediaComponent implements OnInit, OnDestroy {
       let endIndex = firstPartOfString.indexOf("<h2>", 0);
       let startIndex = firstPartOfString.indexOf('</h2>', 0);
       let middleOfString = firstPartOfString.substring(startIndex, endIndex);
-      let finalString = middleOfString.split('href="/wiki').join('target="_blank" href="https://en.wikipedia.org/wiki');
-      finalString.split('title="Edit section: History">edit').join('>');
-      return finalString;
+      let paragraphs = middleOfString.split('<p>');
+      if (paragraphs.length > 6){
+        middleOfString = paragraphs.slice(0,5).join('');
+      }
+      let history = middleOfString.split(/<.*?>/g).join("");
+      return history.split(/&.*?;/g).join("");
```

Comment:
Can we add a unit test to verify that the HTML gets transformed correctly?

Reply:
There is one.

Reply:
In wiki-media.component.spec.ts

Reply:
Oops, I missed that file apologies.
Another question though: it appears that the test 'displays location history' checks that the body is equal to the HTML string of data with tags included.
With this change, that should be failing now because the tags should be all removed from the data before it is placed in the template.